pom.xml

```diff
@@ -37,7 +37,8 @@
     </checkstyle.header.template>
     <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
     <checkstyle.logViolationsToConsole>true</checkstyle.logViolationsToConsole>
-    <jacoco.haltOnFailure>true</jacoco.haltOnFailure>
+    <jacoco.haltOnFailure>false</jacoco.haltOnFailure>
+    <jacoco.line.coveredratio.minimum>0.6</jacoco.line.coveredratio.minimum>
```

Comment:
No need to override the threshold, no one cares as long as it doesn't fail the build, which is prevented by the property one line above.

Reply:
@rsynek 
Copied them from kie-parent - I know it is "ignored" due to the "haltOnFailure"

Reply:
As it's not used, I would recommend removing the property from here.